src/pages/docs/packaging-applications/package-repositories/guides/container-registries/azure-container-services.md

```diff
@@ -21,17 +21,38 @@ Make sure you select **Enable** under the **Admin user** option. This is what wi
 
 With the Admin user toggle enabled, you will be provided with username and password credentials, these will needed these when you create the Octopus Deploy feed. The password can be regenerated at any time so long as you keep your Octopus instance updated with the new credentials.
 
 ## Adding an Azure Container Registry as an Octopus External Feed
-Create a new Octopus Feed (**Library ➜ External Feeds**) and select the `Docker Container Registry` Feed type. With this selected you will need to provide the credentials configured above.
+Create a new Octopus Feed (**Library ➜ External Feeds**) and select the `Azure Container Registry` Feed type. With this selected you will need to provide the username and password credentials configured above.
 
 :::figure
 ![Azure Container Services Registry Feed](/docs/packaging-applications/package-repositories/guides/container-registries/images/azure-feed.png)
 :::
 
 Save and test your registry to ensure that the connection is authorized successfully.
+
+## Adding an Azure Container Registry with OpenID Connect as an Octopus External Feed
+Octopus Server `2025.2` adds support for OpenID Connect to ACR feeds. To use OpenID Connect authentication you have to follow the [required minimum configuration](/docs/infrastructure/accounts/openid-connect#configuration). 
+
+Before creating an OpenID Connect Azure Container Registry feed, you will need an Microsoft Entra ID App Registration and a Federated Credential. 
+
+If you do not currently have an Microsoft Entra ID App Registration follow the [App Registration](https://oc.to/create-azure-app-registration) guide.
+
+To manually create a Federated Credential follow the [Add a federated credential](https://oc.to/create-azure-credentials) section in the Microsoft Entra ID documentation. The federated credential will need the **Issuer** value set to the publicly accessible Octopus Server URI configured, this value must also not have a trailing slash (/), for example `https://samples.octopus.app`.
+
+Create a new Octopus Feed (**Library ➜ External Feeds**) and select the `Azure Container Registry` Feed type. With this selected you will need choose OpenID Connect as the authentication type.
+
+Add the following properties to the feed credentials:
+- **Client ID:** _{{The Azure Active Directory Application ID (Client ID)}}_
+- **Tenant ID:** _{{The Azure Active Directory Tenant ID}}_
+- **Subject:** *Please read [OpenID Connect Subject Identifier](/docs/infrastructure/accounts/openid-connect#subject-keys) on how to customize the **Subject** value*
+- **Audience**  _{{The audience set on the Federated Credential}}_ *This can be set to the default of* `api://AzureADTokenExchange` *or a custom value if needed*
+
+:::div{.warning}
+At this time, OpenID Connect external feeds are not supported for use with Kubernetes containers because the short lived credentials they generate are not suitable for long running workloads.
```

Comment:
Nit: my brain forgets the beginning of this sentence before I finish it... almost definitely a me problem, but heres a suggestion nonetheless 😆 .

```suggestion
At this time, OpenID Connect external feeds are not supported for use with Kubernetes containers. This is because the short lived credentials they generate are not suitable for long running workloads.
```